framework/platform/bpl/uci/cfg/bpl_cfg_helper.cpp

```diff
@@ -49,6 +49,22 @@ int cfg_get_prplmesh_param(const std::string &param, char *buf, size_t buf_len)
     return cfg_uci_get(path, buf, buf_len);
 }
 
+int cfg_get_prplmesh_radio_param(int radio_id, const std::string &radio_param, char *buf,
+                                 size_t buf_len)
+{
+    char path[MAX_UCI_BUF_LEN] = {0};
+
+    if (buf_len > MAX_UCI_BUF_LEN) {
+        buf_len = MAX_UCI_BUF_LEN;
+    }
+
+    if (snprintf_s(path, MAX_UCI_BUF_LEN, "prplmesh.radio%d.%s", radio_id, radio_param.c_str()) <=
```

Comment:
I personally don't like adding '{' for one liners, @morantr thinks otherwise, but whatever you choose please be consistent.
(nitpick)

Reply:
ok